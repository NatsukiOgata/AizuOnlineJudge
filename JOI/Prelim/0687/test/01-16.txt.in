90
293 178 486 65 288 329 451 243 446 95 345 204 374 48 123 124 477 259 58 485 64 75 246 149 156 500 401 240 495 361 330 421 415 434 263 265 353 441 379 268 396 360 284 497 148 274 103 484 411 443 121 98 66 461 321 386 19 217 487 359 99 11 234 475 378 120 327 328 252 410 314 158 104 14 323 398 57 233 471 175 276 150 211 437 219 384 108 212 473 324
