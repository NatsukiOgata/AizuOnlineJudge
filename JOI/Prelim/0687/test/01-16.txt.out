6061
18676
